compiler/rustc_middle/src/query/mod.rs

```diff
@@ -147,7 +147,7 @@ rustc_queries! {
     /// we will have to assume that any change means that you need to be recompiled.
     /// This is because the `hir_crate` query gives you access to all other items.
     /// To avoid this fate, do not call `tcx.hir().krate()`; instead,
```

Comment:
this one doesn't exist anymore either, right?

Reply:
Yup, fixed also.